subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc

```diff
@@ -702,6 +702,55 @@ tasks.create("myTask") {
 // === `Property`, `Provider` and `NamedDomainObjectProvider`
 
 
+[[kotdsl:assignment]]
+== Kotlin Property assignment
```

Comment:
❓ I have trouble with the current way the feature is called. Kotlin already has "property assignment." I feel the capitalized word does not help people understand it's about `Property`, and making it `Property` instead of Property in the title feels weird, and also not too helpful.

Maybe the name for the feature should be "Kotlin lazy property assignment"?